skiplang/prelude/src/skstore/FixedDir.sk

```diff
@@ -205,16 +213,17 @@ class CompactFixedDir protected {
     };
   }
 
-  fun get(i: Int): FixedRow<Array<File>> {
+  fun get(i: Int): FixedRow<Key, Array<File>> {
     compactRow = this.data.unsafe_get(i);
     rowValues = SKDB.RowValues(compactRow.values, compactRow.repeat);
     rowKey = SKDB.RowKey(rowValues, this.metadata.kinds);
     sourceKey = compactRow.sourceKey match {
     | None() -> rowKey
     | Some(otherKey) -> otherKey
     };
+    fixedRowKey: Key = rowKey;
```

Comment:
What's going on here - just helping out the type checker?  Seems like this shouldn't be needed